src/parser/xlsx/utilities/common.util.ts

```diff
@@ -0,0 +1,35 @@
+import * as xlsx from 'xlsx';
+
+
+export default class CommonUtil {
+
+    /**
+     * Checks all required sheets are present in excel file or not
+     * @returns Returns boolean value for whether all required sheets are present in excel file or not
+     */
+    static validateSheetNames(requiredSheetNames: string[], availableSheetNames: string[]) {
+        return Object.values(requiredSheetNames).every((knownSheetName) => availableSheetNames.find((availableSheet: string) => availableSheet?.trim() === knownSheetName?.trim()));
+    }
+
+    /**
+     * Checks if all required column are present in the input sheet or not
+     * @param sheet 
+     * @param {string[]} knownColumnNames 
+     * @returns Returns boolean value for whether all required column are present in the input sheet or not
+     */
+    static validateSheetColumns(sheet: any, knownColumnNames: string[]) {
```

Comment:
Please check if we can send sheet columns only instead of entire sheet because responsibility of this function is to validate sheet names only.